systemd.go

```diff
@@ -78,17 +82,42 @@ func (s *SystemdController) Create(path string, resources *specs.LinuxResources)
 	if err != nil {
 		return err
 	}
+	var once sync.Once
```

Comment:
This should be declared at the package level but otherwise, looks good

Reply:
Ok, fixed